src/runtime_config/actions.js

```diff
@@ -68,6 +68,30 @@ function checkLength(value, min, max, field) {
   }
```

Comment:
These edits need to happen in default_actions.js

Reply:
git auto resolved most of it 👍